dataservice/extensions/flask_indexd.py

```diff
@@ -224,20 +259,25 @@ def delete(self, record):
             return record
 
         if record.rev is None:
-            r = self.session.get(self.url + record.latest_did)
+            try:
+                r = self.session.get(self.url + record.latest_did)
+            except RequestException as err:
+                abort(500, f"Problem getting record in Indexd: {err}")
             record.rev = r.json()['rev']
 
         url = '{}{}?rev={}'.format(self.url, record.latest_did, record.rev)
-        resp = self.session.delete(url)
-        self.check_response(resp)
         try:
+            resp = self.session.delete(url)
+            self.check_response(resp)
             resp.raise_for_status()
         except HTTPError:
```

Comment:
Same question as before...why do we only attach the indexd error on delete errors? Seems like a good thing to do for all requests.